### PR TITLE
Makes GitHub Stop adding that setting file randomly

### PR DESCRIPTION
**Pull request by Vexylius**
## About The Pull Request
https://www.youtube.com/watch?v=13881znnNAU

## Why It's Good For The Game
It's annoying

### CODE REVIEW

**Comment by Sylphily** [comment]
Thank the lord that was annoying as hell

File: .gitignore [hunk 1]
```diff
@@ -196,7 +196,6 @@ Temporary Items
 #Visual studio stuff
 *.vscode/*
 !/.vscode/extensions.json
```
Comment (Fikou) [inline .gitignore]: cut this too

Comment (Vexylius) [reply .gitignore]: if you say so

**Comment by Fikou** [comment]
delete ur changelog

**Comment by Vexylius** [comment]
ok

**Pull request merged**